src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -2209,13 +2211,13 @@ namespace AirflowNetwork {
                     format("{}Errors found getting {} object. Previous error(s) cause program termination.", RoutineName, CurrentModuleObject));
             }
 
-            AirflowNetworkSimu.MaxIteration = static_cast<int>(Numbers(1));
-            AirflowNetworkSimu.RelTol = Numbers(2);
-            AirflowNetworkSimu.AbsTol = Numbers(3);
-            AirflowNetworkSimu.ConvLimit = Numbers(4);
-            AirflowNetworkSimu.Azimuth = Numbers(5);
-            AirflowNetworkSimu.AspectRatio = Numbers(6);
-            AirflowNetworkSimu.MaxPressure = 500.0; // Maximum pressure difference by default
+            simulation_control.maximum_iterations = static_cast<int>(Numbers(1));
+            simulation_control.relative_convergence_tolerance = Numbers(2);
+            simulation_control.absolute_convergence_tolerance = Numbers(3);
+            simulation_control.convergence_acceleration_limit = Numbers(4);
+            simulation_control.azimuth = Numbers(5);
+            simulation_control.aspect_ratio = Numbers(6);
+            simulation_control.MaxPressure = 500.0; // Maximum pressure difference by default
```

Comment:
I'm still using the IDF input style here, this all gets removed in the input refactor.